src/hearings/containers/linked-hearings/linked-hearings.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { select, Store } from '@ngrx/store';
+import * as fromHearingStore from '../../store';
+
+@Component({
+  selector: 'exui-linked-hearingss',
+  templateUrl: './linked-hearings.component.html',
+  styleUrls: ['./linked-hearings.component.scss']
+})
+export class LinkedHearingsComponent {
+  public caseId: string;
+  public caseTitle: string;
+  constructor(private readonly hearingStore: Store<fromHearingStore.State>,
+              private readonly route: ActivatedRoute) {
+    this.caseId = this.route.snapshot.params.caseId;
+    this.hearingStore.pipe(select(fromHearingStore.getHearingsFeatureState)).subscribe(
+      state => {
+        const caseName = state.hearingValues.serviceHearingValuesModel ? state.hearingValues.serviceHearingValuesModel.caseName : '';
+        this.caseTitle = `${caseName} ${this.caseId}`;
```

Comment:
We can pass the caseId, caseTitle to the sub component, and let the sub component to aggregate the caseTitle, as all the caseId needs to show as '1234-5678-9012-3456' with '-', there has be a caseId pipe can transform this in the html.